contracts/deploy/013_three_pool_strategies.js

```diff
@@ -0,0 +1,219 @@
+
+  // On mainnet, the governor is the Timelock contract.
+  let strategyGovernorAddress;
+  if (isMainnet) {
+    strategyGovernorAddress = cMinuteTimelock.address;
+  } else {
+    strategyGovernorAddress = governorAddr;
+  }
+
+  //
+  // Curve USDC Strategy and Proxy
+  //
+  const dCurveUSDCStrategyProxy = await deploy("CurveUSDCStrategyProxy", {
+    from: deployerAddr,
+    contract: "ThreePoolStrategyProxy",
+    ...(await getTxOpts()),
+  });
+  await ethers.provider.waitForTransaction(
+    dCurveUSDCStrategyProxy.receipt.transactionHash,
+    NUM_CONFIRMATIONS
+  );
+  log("Deployed CurveUSDCStrategyProxy", dCurveUSDCStrategyProxy);
+
+  const dCurveUSDCStrategy = await deploy("CurveUSDCStrategy", {
+    from: deployerAddr,
+    contract: "ThreePoolStrategy",
+    ...(await getTxOpts()),
+  });
+  await ethers.provider.waitForTransaction(
+    dCurveUSDCStrategy.receipt.transactionHash,
+    NUM_CONFIRMATIONS
+  );
+  log("Deployed CurveUSDCStrategy", dCurveUSDCStrategy);
+
+  const cCurveUSDCStrategyProxy = await ethers.getContract(
+    "CurveUSDCStrategyProxy"
+  );
+  // Get contract instance through Proxy
+  const cCurveUSDCStrategy = await ethers.getContract(
+    "CurveUSDCStrategy",
+    cCurveUSDCStrategyProxy.address
+  );
+
+  transaction = await cCurveUSDCStrategy
+    .connect(sDeployer)
+    .transferGovernance(strategyGovernorAddress, await getTxOpts());
+  await ethers.provider.waitForTransaction(transaction.hash, NUM_CONFIRMATIONS);
+  log(
+    `CurveUSDCStrategy transferGovernance(${strategyGovernorAddress}) called`
+  );
+
+  //
+  // Curve USDT Strategy and Proxy
+  //
+
+  const dCurveUSDTStrategyProxy = await deploy("CurveUSDTStrategyProxy", {
+    from: deployerAddr,
+    contract: "ThreePoolStrategyProxy",
+    ...(await getTxOpts()),
+  });
+  await ethers.provider.waitForTransaction(
+    dCurveUSDTStrategyProxy.receipt.transactionHash,
+    NUM_CONFIRMATIONS
+  );
+  log("Deployed CurveUSDTStrategyProxy", dCurveUSDTStrategyProxy);
+
+  const dCurveUSDTStrategy = await deploy("CurveUSDTStrategy", {
+    from: deployerAddr,
+    contract: "ThreePoolStrategy",
+    ...(await getTxOpts()),
+  });
+  await ethers.provider.waitForTransaction(
+    dCurveUSDTStrategy.receipt.transactionHash,
+    NUM_CONFIRMATIONS
+  );
+  log("Deployed CurveUSDTStrategy", dCurveUSDTStrategy);
+
+  const cCurveUSDTStrategyProxy = await ethers.getContract(
+    "CurveUSDTStrategyProxy"
+  );
+  // Get contract instance through Proxy
+  const CurveUSDTStrategy = await ethers.getContract(
+    "CurveUSDTStrategy",
+    cCurveUSDTStrategyProxy.address
+  );
+
+  transaction = await CurveUSDTStrategy.connect(sDeployer).transferGovernance(
+    strategyGovernorAddress,
+    await getTxOpts()
+  );
+  await ethers.provider.waitForTransaction(transaction.hash, NUM_CONFIRMATIONS);
+  log(`CurveUSDTStrategy transferGovernance(${strategyGovernorAddress} called`);
+
+  if (!isMainnet && !isRinkeby) {
```

Comment:
Why not running those calls to initialize on Mainnet?

Reply:
👍